capabilities/crypto/exchange-rate/maps/binance.suma

```diff
@@ -0,0 +1,45 @@
+profile = "crypto/exchange-rate@1.0"
+provider = "binance"
+
+"""
+ExchangeRate map
+"""
+map GetExchangeRate {
+  call sendExchangeRateRequest(from = input.from, to = input.to) {
+    return map result if (!outcome.error) {
+      rate = outcome.data
+    }
+  }
+
+  call sendExchangeRateRequest(from = input.to, to = input.from) {
+    return map result if (!outcome.error) {
```

Comment:
@lukas-valenta @TheEdward162 is it considered as bug that `outcome` doesn't contain `error` object when operation fails? Fixing it will be breaking change for already existing maps.